dist/rpm/singularity.spec.in

```diff
@@ -45,130 +45,55 @@ Requires: squashfs
 Requires: squashfs-tools
 %endif
 
-Requires: %{name}-runtime = %{version}-%{release}
+
+Provides: %{name}-runtime
+Obsoletes: %{name}-runtime
 
 %description
 Singularity provides functionality to make portable
 containers that can be used across host environments.
 
-%package devel
-Summary: Development libraries for Singularity
-Group: System Environment/Development
-
-%description devel
-Development files for Singularity
-
-%package runtime
-Summary: Support for running Singularity containers
-Group: System Environment/Base
-
-%description runtime
-This package contains support for running containers created
-by the %{name} package.
-
 %prep
 %setup
 
+mkdir -p gopath/%{singgopath}
+mv `ls -d .??* *|grep -v gopath` gopath/%{singgopath}
 
-%build
-if [ ! -f configure ]; then
-  ./autogen.sh
-fi
+GOTAR=go%{goversion}.linux-amd64.tar.gz
+wget https://dl.google.com/go/$GOTAR
+tar -xf $GOTAR
 
-%configure
-%{__make} %{?mflags}
 
+%build
+export PATH=$PWD/go/bin:$PATH
+export GOPATH=$PWD/gopath
```

Comment:
Should GOROOT be set here to:  `GOROOT=${PWD}/go`  ?

Reply:
I have never had to set GOROOT.  It must default to `` `dirname $0` `` in the go binary.

On the other hand I guess if the user has set it then that would override the default so you're probably right.  I will add it.

Reply:
ok I made that change, and rebased again on the master branch.